starters/tailwind/src/ComboBox.tsx

```diff
@@ -34,7 +35,7 @@ export function ComboBox<T extends object>(
       {description && <Description>{description}</Description>}
       <FieldError>{errorMessage}</FieldError>
       <Popover className="w-[--trigger-width]">
-        <ListBox items={items} className="outline-0 p-1 max-h-[inherit] overflow-auto">
+        <ListBox items={items} className="outline-0 p-1 max-h-[inherit] overflow-auto [clip-path:inset(0_0_0_0_round_.75rem)]">
```

Comment:
I noticed the scrollbar at the bottom of a combobox with sections popup is off by a pixel or two. This might be related to the combobox with sections clipping and the corners not showing with sections which we aren't fixing. 
<img width="220" alt="image" src="https://github.com/adobe/react-spectrum/assets/309540/7f3d4d51-d5d4-4072-9516-d8337897e5cb">
This scrollbar issue is also on calendar and at the top the scrollbar isn't rounded.
<img width="149" alt="image" src="https://github.com/adobe/react-spectrum/assets/309540/dafae6ec-5f34-477d-8dc0-91b3857500be">